### PR TITLE
Manage Users > Create User/Login/Update/Delete

### PR DESCRIPTION
**Pull request by MichaelMicrof**
[JN_35](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-35) Create user
[JN_36](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-36) Login
[JN_37](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-37) Check People Page
[JN_38](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-38) Delete user
[JN_41](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-41) Update fullname
[JN_64](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-64) Create user with name "_"
[JN_72](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-72) Create user with name "-"
[JN_79](https://irynab.atlassian.net/jira/software/projects/JN/boards/1/backlog?selectedIssue=JN-79) Create user with long name

### CODE REVIEW

**Review by Jenny-ai — APPROVED** [review]

File: tests/test_manage_user.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        verify, that user present on the page
+        :return:
+        """
+        driver = ManageUserPage(self.driver)
+        driver.go_to_page(ManageUserPage.URL_USER_MANAGE)
+        driver.click_button_create_new_user()
+        driver.fill_all_field_and_click_save(ManageUserPage.USER_NAME_HYPHEN, ManageUserPage.PASSWORD)
+
+        assert driver.get_elements_text(ManageUserPage.USER_ID_HYPHEN)[0] == ManageUserPage.USER_NAME_HYPHEN
+
```
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply tests/test_manage_user.py:93]: фактически они нужны отдельно

File: tests/test_manage_user.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        driver = ManageUserPage(self.driver)
+        driver.go_to_page(ManageUserPage.URL_USER_MANAGE)
+        driver.click_button_create_new_user()
+        driver.fill_all_field_and_click_save(ManageUserPage.USER_NAME, ManageUserPage.PASSWORD)
+
+        assert driver.get_current_url() == ManageUserPage.URL_USER_MANAGE
+        assert driver.get_elements_text(ManageUserPage.USER_ID)[0] == ManageUserPage.USER_NAME
+
+    def test_added_new_user_on_the_people_page(self):
+        """
```
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply tests/test_manage_user.py:29]: они отличаются

File: tests/test_manage_user.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+        assert driver.get_current_url() == TD.BASE_URL
+
+    def test_added_new_user_on_the_people_page(self):
+        """
+        TC_JN_37
+        verify that on the PeoplePage new record in the list
+        :return:
+        """
+        driver = PeoplePage(self.driver)
+        driver.go_to_page(URLLocators.URL_PEOPLE)
```
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply tests/test_manage_user.py:61]: нет, еще надо делать переход

File: pages/ManageUserPage.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    def click_button_create_new_user(self):
+        """
+        push on the button
+        :return:
+        """
+        self.click(self.CREATE_USER)
+        return self
+
+    def fill_all_field_and_click_save(self, name, password):
+        """
```
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply pages/ManageUserPage.py:68]: ок

File: pages/LoginPage.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    def login_jenkins(self, login, password):
+        """
+        input login/password and push button Submit on the Login Page
+        :param login:
+        :param password:
+        :return:
+        """
+        self.do_send_keys(LoginPage.INPUT_LOGIN, login)
+        self.do_send_keys(LoginPage.INPUT_PASSWORD, password)
+        self.click(LoginPage.SIGN_IN_BUTTON)
```
[the remark this answers is not in the document]
Comment (MichaelMicrof) [reply pages/LoginPage.py:27]: ок

**Pull request merged**